test/Chainweb/Test/Pact/PactMultiChainTest.hs

```diff
@@ -209,6 +210,20 @@ chainweb213Test = do
         , "(create-table tbl)"
         ]
 
+pact45UpgradeTest :: PactTestM ()
+pact45UpgradeTest = do
+  runToHeight 54
```

Comment:
This is more an issue with `ChainwebVersion` than with the forks per se.  Ideally, tests would be able to mock their own `ChainwebVersion` and set forks for its own requirements (perhaps by implementing a typeclass that specifies all the fork points). 

As it is, our approach suffers from a great deal of coupling and complexity, including abusing different version graphs to get "sub-versions" so that tests don't clobber each other.

Finally, it is why it is hard to do something like "just get the fork block height", as we are instead usually wanting some kind of particular test (see `AtOrAfter` in some of the fork functions). 

Agree with @jmcardon that this is out of scope for this PR.

Reply:
Coming soon (TM)